app/src/test/java/net/bible/service/format/osistohtml/taghandler/NoteHandlerTest.java

```diff
@@ -71,7 +82,7 @@ public void simpleNote() {
 		noteHandler.end();
 		writer.write("after note");
 		
-		assertThat(writer.getHtml(), equalTo("before note<span class='noteRef'>a</span> after note"));
+		assertThat(writer.getHtml(), equalTo(String.format("before note<a href='%s:%s%d/a' class='noteRef'>a</a> after note", Constants.NOTE_PROTOCOL, basisRef.getOsisID(), verseInfo.currentVerseNo)));
```

Comment:
In tests, you shouldn't do like that. Let's say, you accidentally change Constants.NOTE_PROTOCOL. The test should fail, but it does not. Tests should be broken (and fixed) if such changes are made, accidentally or not.

Reply:
I disagree, if the note protocol constant is changed, the functionality will still be the same, so the test should not fail.